deploy

```diff
@@ -2,10 +2,17 @@
 #! nix-shell ./shell.nix -i bash
 set -euo pipefail
 
+options=(
+  --option extra-substituters "https://nix-community.cachix.org"
+  --option binary-cache-public-keys "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs="
```

Comment:
I don't know why but nixops seems to ignore those options

Reply:
Is your user trusted?

Reply:
Yeah. Do you mind if the PR gets merged regardless? I don't think it hurts in its current state.

Reply:
I don't mind.

Reply:
thanks! :)